frontend-service-client/src/main/java/com/amazonaws/blox/BloxClientBuilder.java

```diff
@@ -33,6 +35,53 @@
     }
 
     /**
+     * Specify an implementation of {@link AWSCredentialsProvider} to be used when signing IAM auth'd requests
+     *
+     * @param iamCredentials
+     *        the credential provider
+     */
+    @Override
+    public void setIamCredentials(AWSCredentialsProvider iamCredentials) {
```

Comment:
That's okay. Just trying to figure out what I need to review and it looks like everything in fe client is auto generated right?